Formula/osslsigncode.rb

```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2549903650fdd0189c1083a177af1c789364758080828b8ea6cf46c376ed6d65"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "openssl@1.1"
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
+
   def install
-    system "./bootstrap"
-    system "./configure", *std_configure_args
-    system "make", "install"
+    # Fixing libcurl linker issue with `curl_version`
```

Comment:
What is the linker issue? Has it been reported upstream?

Reply:
Ping?

Reply:
could not find the `curl_version`

Reply:
build failure

```
  "_curl_easy_strerror", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_slist_append", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_slist_free_all", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_version", referenced from:
      _main in osslsigncode.c.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
reported, https://github.com/mtrojnar/osslsigncode/issues/179